CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## [Unreleased]
```

Comment:
Nice with the cleanups in the changelog!

If you want to keep it really tidy, you could keep consistent line endings as well (i e not mixing ending with period and not ending with period)